### ISSUE TITLE
[Alerting] fix commented out tests

### ISSUE THREAD

**Issue by kavilla** [opensearch-project/opensearch-dashboards-functional-test#183]
Origin: https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/182

The tests were commented in alerting plugin:
https://github.com/opensearch-project/alerting-dashboards-plugin/pull/214

So matching the source repo, the tests step were commented as well. We will need to copy-paste the fixes from the source repo.

**Comment by tianleh** [comment]
@AWSHurneyt Do you plan to implement this as part of `2.0.0` release?

**Comment by AWSHurneyt** [comment]
I don't believe so. As destinations are being deprecated, my understanding at this time is that we plan to refactor the destination-related portions of these tests to use notification channels instead. 

Including @[qreshi](https://github.com/qreshi), as he's been working on the notifications plugin.

**Comment by kavilla** [comment]
yes I actually have a PR running right now https://github.com/opensearch-project/opensearch-dashboards-functional-test/pull/191, then removed some of that stuff but not all. it does seem all the destinations stuff relies on notifications repo which seems like even more reason to onboard on to this repo. If the expected functionality of the alerting plugin relies on the notifications plugin, then having the sanity tests only run from the notifications repo where it can assume that it's the only plugin installed doesn't reflect reality.
